### DISCUSSION TITLE
希望能提供容灾功能

### DISCUSSION THREAD

**Discussion by dbbao** [apache/cloudberry#559] [Ideas / Feature Requests]
### Description

希望能提供容灾功能

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

**Comment by my-ship-it** [comment]
可以提供一下具体的场景

**Comment by tuhaihe** [comment]
Hi @dbbao could you help rewrite your feature request in English? So that more global community members can join your discussions. Thanks.

**Comment by my-ship-it** [comment]
Actually the topic is under discussion now: https://github.com/orgs/cloudberrydb/discussions/597
